seqr/views/utils/file_utils.py

```diff
@@ -86,25 +94,25 @@ def save_uploaded_file(request, process_records=None):
         raise ValueError("Received %s files instead of 1" % len(request.FILES))
 
     # parse file
-    stream = request.FILES.values()[0]
+    stream = list(request.FILES.values())[0]
     filename = stream._name
 
     json_records = parse_file(filename, stream)
     if process_records:
         json_records = process_records(json_records, filename=filename)
 
     # save json to temporary file
-    uploaded_file_id = hashlib.md5(str(json_records)).hexdigest()
+    uploaded_file_id = hashlib.md5(str(json_records).encode('utf-8')).hexdigest()
```

Comment:
I don't have any problem with this, but I'm curious why it is neccessary

Reply:
It is because `json_records` contains unicode and hash function requires unicode must be encoded. There is no issue before maybe because `json_records` contains ascii only.